src/network/networkd-ipv6ll.c

```diff
@@ -219,7 +220,7 @@ int link_set_ipv6ll_stable_secret(Link *link) {
         }
 
```

Comment:
add assert(link->manager) at the top, here and elsewhere below

Reply:
I can't find any `link->manager` usage here, maybe you meant in

Reply:
it's dereferenced in the line below:

```
        return sysctl_write_ip_property(AF_INET6, link->ifname, "stable_secret",
                                        IN6_ADDR_TO_STRING(&a));
                                        IN6_ADDR_TO_STRING(&a), &link->manager->sysctl_shadow);
```

Reply:
Sorry, I was looking at the wrong patch, apparently GH show some comments on all commits